### PR TITLE
feature/6804-Dylan-RXListEnhancementCopy

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
Updated copy for RX List header and the RX help screen

## Screenshots/Video
Toggle: <details><summary>Before/after:</summary>&nbsp;&nbsp;</details>
Toggle: <details><summary>Before/after:</summary>&nbsp;&nbsp;</details>
Toggle: <details><summary>Before/after:</summary>&nbsp;&nbsp;</details>

## Testing
yarn test

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
Update the body copy on the list view to now read: This list only shows prescriptions filled by VA pharmacies and may not include all your medications.
 Update the help large panel content to now show the following bulleted information
Bullet 1: New prescriptions not yet processed by a VA pharmacy
Bullet 2: Prescriptions filled at non-VA pharmacies
Bullet 3: Prescriptions that are inactive for more than 180 days
Bullet 4: Medications administered at a clinic or ER (existing)
Bullet 5: Self-entered medications (existing)
 Update the blue alert panel that is triggered when a successful call to fetch user's prescriptions, but the API call did not return any (i.e. user has no prescriptions) with the updated bulleted list shown in AC 2.

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx [hunk 1]
```diff
@@ -21,11 +21,25 @@ const PrescriptionHistoryNoPrescriptions: FC = () => {
     mb: theme.dimensions.contentMarginBottom,
   }
 
-  const bullets: string[] = [
-    t('prescriptions.notFound.bullets.one'),
-    t('prescriptions.notFound.bullets.two'),
-    t('prescriptions.notFound.bullets.three'),
-    t('prescriptions.notFound.bullets.four'),
+  const medicationNoIncludedList: Array<VABulletListText> = [
```
Comment (theodur) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx:24]:
```suggestion
  const medicationsNotIncludedList: Array<VABulletListText> = [
```
Typo

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx [hunk 2]
```diff
@@ -35,7 +49,7 @@ const PrescriptionHistoryNoPrescriptions: FC = () => {
           <TextView pt={theme.paragraphSpacing.spacing20FontSize} paragraphSpacing={true} accessibilityLabel={a11yLabelVA(t('prescriptions.notFound.yourVA'))}>
             {t('prescriptions.notFound.yourVA')}
           </TextView>
-          <VABulletList listOfText={bullets} paragraphSpacing={true} />
+          <VABulletList listOfText={medicationNoIncludedList} paragraphSpacing={true} />
```
Comment (theodur) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx:52]: This wasn't caused by the changes in this PR, but I noticed the spacing between the `Medications not included` text and the bullet point list is much greater than in the Figma mock

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx (hunk 1 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx]: fixed

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx (hunk 2 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx]: Fixed the spacing to be compliant with WCAG

**Review by dumathane — DISMISSED** [review]

**Review by theodur — APPROVED** [review]
It looks like the spacing discrepancies still aren't fixed. I think the `paragraphSpacing` needs to be replaced with `mb={theme.dimensions.condensedMarginBetween}` [here](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/feature/6804-Dylan-RXListEnhancementCopy/VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHelp/PrescriptionHelp.tsx#L52). Also, the spacing under the bullet list is still more than what it is in the mock.

Since these issues are unrelated to the changes in this PR I'm gonna approve still. Those can either be fixed in a new commit or a new ticket can be made for that

**Review by theodur — DISMISSED** [review]

**Review by theodur — DISMISSED** [review]

**Review by theodur — APPROVED** [review]

**Comment by TKDickson** [comment]
@Sparowhawk please don't actually merge this until we're certain that we have the 2.14 release ironed out; FYI @dumathane

**Pull request merged**